Sources/SwiftRepo/SwiftUI/LoadingControllerView.swift

```diff
@@ -65,8 +66,14 @@ public struct LoadingControllerView<DataType, Content, LoadingContent, ErrorCont
         }
         // Make this view greedy so that it occupies the same space across all loading states.
         .frame(maxWidth: .infinity, maxHeight: .infinity)
-        // This keeps animations together if new animations are created while other animations are in progress.
-        .geometryGroup()
+        .viewAsArgument { view in
+            if #available(iOS 17, *) {
+                // This keeps animations together if new animations are created while other animations are in progress.
+                view.geometryGroup()
+            } else {
+                view
+            }
```

Comment:
Is not clear to me how the animations would look like if we don't apply `geometryGroup()` and new animations are created. But probably we can leave that to a separate PR, in case the animations are unacceptable.

Reply:
We can define our own `geometryGroup()` for pre-iOS 17 that applies the identify transform, which has the same effect.